src/types/IstioConfigDetails.ts

```diff
@@ -42,7 +42,8 @@ export interface IstioConfigDetails {
   quotaSpecBinding: QuotaSpecBinding;
   policy: Policy;
   meshPolicy: Policy;
-  clusterRbacConfig: ClusterRbacConfig;
+  clusterRbacConfig: RbacConfig;
```

Comment:
I'd maintain the ClusterRbacConfig type here, they have same fields, but conceptually they are different entities.
I think it makes sense to separate them logically, if not it can point to some mistake.

Reply:
wasn't RBacConfig deprecated? https://github.com/istio/istio/issues/8825
and yes, I'd maintain them separately

Reply:
@abonas in 1.1.0 both resources are leaving. The final deprecation will be in a subsequent step.
In any case, latest version of Istio now is 1.0.6 with RbacConfig (no ClusterRbacConfig).

Reply:
Aren't we supporting only the newer version (1.1.0) ?

Reply:
Good question.
There is a JIRA for that https://issues.jboss.org/browse/KIALI-1023
As those types are valid and used for 1.0 and 1.1 we thought that we could do the effort to support both scenarios.
But yes, theorically we should support 1.1 (newer) versions